cluster-install/install.sh

```diff
@@ -104,6 +105,11 @@ readonly __FATAL=50
 #     captured_stdout="$(curl -sS https://bad-url.com 2> "$_ERR_FILE")"
 #     captured_stderr="$(get_captured_err)"
 readonly __ERR_FILE="tmp_last_captured_error.txt"
+
+# __TMP_ENV is used to store the user's env vars so that we can then re-apply them after having sourced
+# their config file. This allows us to make command line vars take precedence over what's in the config file.
+readonly __TMP_ENV="tmp.env"
```

Comment:
As the comment says, this variable will temporarily hold the user's environment variables and reverse the order of priority between command-line vars and config_file vars.

Previously, the order of operations was this:
1. inherit user's environment variables + those specified via the command line
2. apply the config file variables, possibly stomping over those from #1

Now, the order is:
1. (same as previous -- inherit env from user and CLI)
2. store all current environment variables, i.e. those we got in step 1
3. apply the config file variables
4. re-apply the env variable stored in step 2

Note that there's no way to differentiate environment variables provided via the command line VS those that were already set in the shell before the script was called, so it's still not as "ideal" as having proper script options (i.e. `--my-var=something`), but it's one step closer.